targeteditor/src/main/scala/explore/targeteditor/SimbadSearch.scala

```diff
@@ -0,0 +1,46 @@
+package explore.targeteditor
+
+import scala.concurrent.duration._
+
+import cats.data.Validated
+import cats.effect._
+import cats.implicits._
+import eu.timepit.refined.types.string.NonEmptyString
+import fs2._
+import lucuma.catalog.VoTableParser
+import lucuma.core.enum.CatalogName
+import lucuma.core.model.Target
+import sttp.client3._
+
+object SimbadSearch {
+  def search(term: NonEmptyString)(implicit cs: ContextShift[IO]): IO[Option[Target]] = {
+    val backend  = FetchBackend()
+    def response = basicRequest
+      .post(
+        uri"https://simbad.u-strasbg.fr/simbad/sim-id?Ident=${term}&output.format=VOTable"
+      )
+      .readTimeout(5.seconds)
+      .send(backend)
+
+    IO.fromFuture(IO(response))
```

Comment:
This is very very minor, but I think it would be clearer to me if this was `val response = IO(.....send(backend))` and then `IO.fromFuture(response)`.

Reply:
> This is very very minor, but I think it would be clearer to me if this was `val response = IO(.....send(backend))` and then `IO.fromFuture(response)`.

I think that would also be more correct. Because Futures evaluate eagerly, you're supposed to wrap the creation of the Future in IO to suspend it's creation. I think that's why `IO.fromFuture` takes an `IO[Future]` instead of just a `Future`, isn't it?